### DISCUSSION TITLE
Slayer Plugin toggle for not Highlighting Bosses

### DISCUSSION THREAD

**Discussion by verafeiyd** [runelite/runelite#18043] [Ideas]
As the title says, due to NPC Highlight, the current Slayer "Highlight" overrides the Tile Indicator/Highlight of the boss, even if they are using two different modes. Id like to see just a simple "prevent highlighting bosses" for the slayer plugin to fix this, as when youre on a slayer task for a boss you usually have marked, it messes up your setup

**Comment by geheur** [comment]
what if it was just "don't have slayer plugin mark things that something else already marks"? That way no one has to curate a slayer boss list.

**Comment by verafeiyd** [comment]
Ive talked a while ago quite a bit about a priority system with the discord, and through all of the complications that could create with other plugins fighting for priority, a simple "disable overlay on boss slayer versions" was the cleanest way to do it. this is not "curating a boss list", its a toggle to disable slayer from highlighting *any* bosses to let NPC highlight do what it needs to and srguably, the better plugin to do it from
